rollup-http/rollup-http-server/src/http_service.rs

```diff
@@ -48,8 +48,10 @@ pub fn create_server(
         let data = Data::new(Mutex::new(Context {
             rollup_fd: rollup_fd.clone(),
         }));
+        let json_cfg = JsonConfig::default().limit(5242880);
```

Comment:
Looks good to me.
A more canonical way to "say" 5MBi is:

```suggestion
        let json_cfg = JsonConfig::default().limit(5 << 20);
```

Reply:
The rollups device is limited to 2MB, right?
Shouldn't it be limited to 2MB then?

Reply:
> The rollups device is limited to 2MB, right?
> Shouldn't it be limited to 2MB then?

2MB is the limit when dealing with binary data but this 5MB is for json encoded messages.
It makes sense for this to be larger than the original buffer.
Hex encoding for instance will 2x data size for instance.